Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM drujensen/crystal:0.23.1
+FROM crystallang/crystal:0.23.1
 
-WORKDIR /app/user
+# Install Dependencies
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -qq && apt-get install -y --no-install-recommends libpq-dev libsqlite3-dev libmysqlclient-dev libreadline-dev git curl vim netcat
 
-ADD . /app/user
+# Install Node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs
 
-RUN crystal deps
+WORKDIR /opt/amber
+
+# Build Amber
+ENV PATH /opt/amber/bin:$PATH
+ADD . /opt/amber
+RUN shards build amber
 
 CMD ["crystal", "spec", "-D", "run_build_tests"]
```

Comment:
Now we have one docker image that contains crystal and amber.  It is the same image we use in travis that the tests are run against.